src/scripts/main.js

```diff
@@ -1,7 +1,31 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${type}`;
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const h2 = document.createElement('h2');
+
+  h2.className = 'title';
+  h2.textContent = title;
+  h2.style.whiteSpace = 'nowrap';
+  notification.appendChild(h2);
+
+  const p = document.createElement('p');
+
+  p.textContent = description;
+  p.innerHTML = description.replace(/\n/g, '<br>');
```

Comment:
There is a redundancy here. You are setting `p.textContent` and then immediately overwriting it with `p.innerHTML`. Consider removing the `p.textContent = description;` line if you intend to use HTML formatting with line breaks.